### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by GretaCB**
- [x] Add CC0 license. I used [gitignore's exact version](https://github.com/github/gitignore/blob/master/LICENSE).
- [x] Add license and contribution info to readme

cc @springmeyer @mapsam

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +70,13 @@ make coverage
 ```
 
 For more details about what `make coverage` is doing under the hood see https://github.com/mapbox/cpp#code-coverage
+
+# Contributing and License
+
+Contributors are welcome! :sparkles: This repo exists as a place to gather C++/Node Addon knowledge that will benefit the larger community. Please contribute your knowledge if youd like.
```
Comment (springmeyer) [inline README.md]: `youd` -> `you'd`

**Comment by GretaCB** [comment]
Thanks for the catch @springmeyer 👍
 
Also, pinging @kathleenlu09 as general FYI

**Comment by mapsam** [comment]
Some README art and badges if you're wanting them, @GretaCB!




```
![readme image](https://mapbox.s3.amazonaws.com/cpp-assets/node-cpp-skel-readme_blue.png)
```

And the badge for others to use

[badge](https://github.com/mapbox/node-cpp-skel)

```
[![badge](https://mapbox.s3.amazonaws.com/cpp-assets/node-cpp-skel-badge_blue.svg)](https://github.com/mapbox/node-cpp-skel)
```

**Comment by mikemorris** [comment]
That skeleton is adorable!

**Pull request merged**